packages/babel-parser/src/parser/comments.js

```diff
@@ -38,6 +38,48 @@ export default class CommentsParser extends BaseParser {
     this.state.leadingComments.push(comment);
   }
 
+  adjustCommentsAfterTrailingComma(node: Node, elements: Node[]) {
+    if (elements.length === 0) {
+      return;
+    }
+    if (this.state.leadingComments.length === 0) {
+      return;
+    }
+    if (!this.state.commentPreviousNode) {
```

Comment:
I think `this.state.commentPreviousNode` will always be non-empty as long as `elements` is not empty.

Reply:
Oh yeah good point.